hw2/SimpleAuthorization/SimpleAuthorization/Authorization.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using System.Security.Authentication;
+
+namespace SimpleAuthorization
+{
+    public class Authorization
+    {
+        private static Dictionary<string, string> authDictionary;
+
+        public static UserModel Run(string[] args)
```

Comment:
Run seems to be not propriate name for this method

Reply:
Renamed to Authorize()